adal/src/main/java/com/microsoft/aad/adal/DefaultTokenCacheStore.java

```diff
@@ -216,7 +216,7 @@ public void removeAll() {
         Map<String, String> results = (Map<String, String>)mPrefs.getAll();
 
         // create objects
-        ArrayList<TokenCacheItem> tokens = new ArrayList<>(results.values().size());
+        List<TokenCacheItem> tokens = new ArrayList<>(results.values().size());
```

Comment:
make sure we have consistency in the change, if you already start to make variable final whenever possible :)